src/ft2font_wrapper.cpp

```diff
@@ -524,6 +524,31 @@ PyFT2Font_set_size(PyFT2Font *self, double ptsize, double dpi)
     self->x->set_size(ptsize, dpi);
 }
 
+const char *PyFT2Font_set_features__doc__ = R"""(
+    Set the font feature tags used for the font.
+
+    Parameters
+    ----------
+    features : tuple[str, ...]
+        The font feature tags to use for the font.
+
+        Available font feature tags may be found at
+        https://learn.microsoft.com/en-us/typography/opentype/spec/featurelist
+)""";
+
+static void
+PyFT2Font_set_features(PyFT2Font *self, py::object features_obj)
```

Comment:
I think you can just take features_obj as a `std::vector<std::string>` unless you really want to restrict to taking tuples?